critiquebrainz/frontend/external/external_players.py

```diff
@@ -0,0 +1,29 @@
+from critiquebrainz.frontend.external import musicbrainz
+import re
+import requests
+import yaml
+
+
+def get_url(mbid):
+    all_url_rels = musicbrainz.get_url_rels_from_releases(musicbrainz.browse_releases(release_group=mbid,
+                                                                                      includes=['url-rels']))
+    streaming_url_rels = {'soundcloud': None, 'bandcamp': None}
+    for url_rel in all_url_rels:
+        if url_rel['type-id'] == '08445ccf-7b99-4438-9f9a-fb9ac18099ee':  # Type-id for streaming music
+            if re.match(r'^(http|https)://soundcloud.com', url_rel['target']):
+                streaming_url_rels['soundcloud'] = url_rel['target']
+            elif re.match(r'^(http|https)://[A-Za-z0-9]+.bandcamp.com/?', url_rel['target']):
+                streaming_url_rels['bandcamp'] = bandcamp_parser(url_rel['target'])
+    return streaming_url_rels
+
+
+def bandcamp_parser(bandcamp_link):
+    """Gives the unique bandcamp id of the entered album/release
+    Required for embedding the player.
+    """
+    html = requests.get(bandcamp_link).text
```

Comment:
I think because it slows down the page load. When I was monitoring my network, i saw it takes upto 3s for the initial page load the initial html of the page(which probably includes this request)